R/fitNullModel.R

```diff
@@ -193,3 +221,83 @@ nullModelSmall <- function(null.model) {
 isNullModelSmall <- function(null.model) {
     is.null(null.model$cholSigmaInv)
 }
+
+
```

Comment:
This probably needs to be updated for GxE tests. The string is more complicated I think.

Reply:
I think the *null model* string should be the same though. The actual model strings are (ignoring random effects):
- null model string: y ~ x1 + x2 + x3
- regular test: y ~ x1 + x2 + x3 + snp
- GxE: y ~ x1 + x2 + x3 + x1*snp

but in all cases, the null model string is just `y ~ x1 + x2 + x3`. You also use the same null model object to run the regular test as well as the GxE test, right?

Reply:
Oh. You are absolutely correct. Sorry about that.